contracts/utils/ExtendedMulticall.sol

```diff
@@ -36,4 +36,40 @@ contract ExtendedMulticall is Multicall {
     function getBlockBasefee() external view returns (uint256) {
         return block.basefee;
     }
+
+    struct Result {
+        bool success;
+        bytes returnData;
+    }
+
+    event FailedDelegatedcall(
+        bytes data,
+        uint256 timestamp,
+        string errorMessage,
+        address indexed sender
+    );
+
+    function tryMulticall(bytes[] calldata data)
+        external
+        returns (Result[] memory results)
+    {
+        results = new Result[](data.length);
+        for (uint256 i = 0; i < data.length; i++) {
+            (bool success, bytes memory result) = address(this).delegatecall(
+                data[i]
+            );
+            if (!success) {
+                // We do not bubble up the revert string from `callData`
+                emit FailedDelegatedcall(
+                    data[i],
+                    block.timestamp,
+                    "low-level delegate call failed",
+                    msg.sender
+                );
```

Comment:
I don't really know if this event adds any value but I added it to be consistent with [AirnodeProtocol](https://github.com/api3dao/airnode-protocol-v1/blob/main/contracts/protocol/AirnodeProtocol.sol#L151-L157) (it can be removed if not necessary)

Reply:
I would remove

Reply:
Related https://github.com/api3dao/airnode-protocol-v1/issues/26